scene/resources/rectangle_shape_2d.cpp

```diff
@@ -47,11 +47,27 @@ Vector2 RectangleShape2D::get_size() const {
 }
 
 void RectangleShape2D::draw(const RID &p_to_rid, const Color &p_color) {
-	RenderingServer::get_singleton()->canvas_item_add_rect(p_to_rid, Rect2(-size / 2, size), p_color);
+	// Draw an outlined rectangle to make individual shapes easier to distinguish.
+	Vector<Vector2> stroke_points;
+	stroke_points.resize(5);
+	stroke_points.write[0] = -size * 0.5;
+	stroke_points.write[1] = Vector2(size.x, -size.y) * 0.5;
+	stroke_points.write[2] = size * 0.5;
+	stroke_points.write[3] = Vector2(-size.x, size.y) * 0.5;
+	stroke_points.write[4] = -size * 0.5;
+
+	Vector<Color> stroke_colors;
+	stroke_colors.resize(5);
+	for (int i = 0; i < 5; i++) {
+		stroke_colors.write[i] = (p_color);
+	}
+
+	RenderingServer::get_singleton()->canvas_item_add_rect(p_to_rid, Rect2(-size * 0.5, size), p_color);
+	RenderingServer::get_singleton()->canvas_item_add_polyline(p_to_rid, stroke_points, stroke_colors);
```

Comment:
I think this can be replaced simply by `draw_rect()`.

Reply:
@KoBeWi There's no `draw_rect()` in RenderingServer.

Reply:
Ah nevermind, I think I confused it with CollisionShape2D.